Library/Homebrew/cmd/tap.rb

```diff
@@ -28,14 +32,16 @@ def install_tap user, repo
     tapd.find_formula { |file| files << file }
     puts "Tapped #{files.length} formula#{plural(files.length, 'e')} (#{tapd.abv})"
 
-    if private_tap?(repouser, repo) then puts <<-EOS.undent
-      It looks like you tapped a private repository. To avoid entering your
-      credentials each time you update, you can use git HTTP credential caching
-      or issue the following command:
+    unless clone_target
```

Comment:
Do we still want to print some form of message if the user appears to be tapping a private repository with a custom URL? Or trust the user to know what to do?